keep/providers/amazonsqs_provider/amazonsqs_provider.py

```diff
@@ -27,15 +27,15 @@ class AmazonsqsProviderAuthConfig:
 
     access_key_id: str = dataclasses.field(
         metadata={
-            "required": True,
-            "description": "Access Key Id",
+            "required": False,
+            "description": "Access Key Id (Leave empty if using IAM role at EC2)",
```

Comment:
Doesn’t it require default because the annotation is str?

Reply:
fixed!